examples/pipe/dingo.ini

```diff
@@ -0,0 +1,129 @@
+# sampling-frequency={{ production.meta['quality']['sample-rate'] }}
+psd-length= 128 #{{ production.meta['quality']['psd-length'] }}
+# psd-maximum-duration=1024
+# psd-method=median
+# psd-start-time=None
+# maximum-frequency=1024
+# minimum-frequency={{ production.meta['quality']['reference-frequency'] }}
+# tukey-roll-off=0.4
+# resampling-method=lal
+
+################################################################################
+## Injection arguments
+################################################################################
+
+# injection-dict=None
+# injection-file=None
+
+################################################################################
+##  Job submission arguments
+################################################################################
+
+accounting = ' ' # If running on a LIGO cluster, add accounting tag here
+# accounting-user = ""
+label={{ production.name }} # GW150914
+local = False
+# local-generation=False
+# local-plot=False
+outdir={{ production.rundir }} # outdir_GW150914
+#overwrite-outdir=True
+request-disk=5.0
+request-memory=32.0
+# request-memory-generation=None
+request-cpus=8
+request-cpus-importance-sampling = 64
+sampling-requirements = [TARGET.CUDAGlobalMemoryMb>40000]
+extra-lines = None
+simple-submission = True
+# conda-env=None
+# scheduler=condor
+# scheduler-args=None
+# scheduler-module=None
+# scheduler-env=None
+# scheduler-analysis-time
+# submit
+# condor-job-priority=0
+# transfer-files=False
+# log-directory=None
+# osg=False
+# desired-sites=None
+# analysis-executable=None
+# analysis-executable-parser=None
+
+################################################################################
+## Output arguments
+################################################################################
+# plot-corner
+# plot-weights
+# plot-log-probs
+# create-summary
+# email
+# notification
+# queue
+# existing_dir
+# webdir = /home/pe.o3/public_html/LVC/o3b-catalog/{{ production.event.name }}/{{ production.name }}
+# summarypages-arguments=None
+# result-format=hdf5
+# final-result
+# final-result-nsamples=20000
+
+################################################################################
+## Prior arguments
+################################################################################
+# prior-dict = None
+
+################################################################################
+## Waveform arguments
+################################################################################
+
+# reference-frequency = None
+# waveform-approximant = None
+
+################################################################################
+##  Sampler arguments
+################################################################################
+
+model = /is/projects/dingo-data/Papers/01_importance_sampling/01_data/01_models/model_XPHM_dp03.pt
+model-init = /is/projects/dingo-data/Papers/01_importance_sampling/01_data/01_models/model_XPHM_dp03_init.pt
```

Comment:
This will have to be overwritten. Can we reference new keys of `production`? I'm not sure how it's supposed to work.

Reply:
I don't know if this is supported by Asimov

Reply:
But since we don't automatically detect which network to use, specifying the network in the settings file isn't really a problem